tables/passive_npc.yaml

```diff
@@ -0,0 +1,44 @@
+name:
+  # escape brackets
+  - '[\w\-]+ [\w\-]+ spirit'
```

Comment:
Nit: I would prefer to alphabetize these entries so it is easier for humans to debug and look up existing values.